tests/FSharp.Collections.ParallelSeq.Tests/FSharp.Collections.ParallelSeq.Tests.fsproj

```diff
@@ -3,8 +3,6 @@
   <PropertyGroup>
     <OutputType>Library</OutputType>
     <TargetFramework>netcoreapp2.0</TargetFramework>
-    <DebugType>pdbonly</DebugType>
-    <Optimize>true</Optimize>
```

Comment:
Any particular reason to remove the optimize?

Reply:
Nope, I will add it back. It just escaped with `<DebugType>pdbonly</DebugType>`.